droidlet/interpreter/robot/tasks.py

```diff
@@ -5,7 +5,9 @@
 import time
 import logging
 import numpy as np
-
+import os
+import math
+from droidlet.memory.robot.loco_memory import DetectedObjectNode
```

Comment:
this isn't in loco_memory_nodes.py ?  when did that change happen?  has this been tested?

Reply:
Good point. Turns out loco_memory imports all loco_memory_nodes (https://github.com/facebookresearch/fairo/blob/main/droidlet/memory/robot/loco_memory.py#L11) which is why this works. Changing this to specifically import from nodes.